src/index.js

```diff
@@ -34,12 +39,14 @@ const addList = () => {
     taskText.addEventListener('change', () => {
       if (taskText.value.length > 0) {
         item.description = taskText.value;
-        stat.saveLocal(list);
+        task.saveLocal(list);
       }
     });
     taskElement.appendChild(taskText);
     const dragIcon = document.createElement('button');
     dragIcon.classList = 'far fa-trash-alt deleteBtn';
+    // const dragIcon = document.createElement('i');
+    // dragIcon.classList = 'fas fa-ellipsis-v drag icon';
```

Comment:
- [ optional ] Hey. nice job. As part of JS best practice, I'll recommend removing unnecessary commented code.